### PR TITLE
feat(react/notification): implement

### PR DESCRIPTION
**Pull request by Kuanisinajar**

### CODE REVIEW

File: packages/core/src/notification/notification.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+} as const;
+
+export const notificationClasses = {
+  host: notificationPrefix,
+  severity: (severity: NotificationSeverity) => `${notificationPrefix}--${severity}`,
+  closeIcon: `${notificationPrefix}__close-icon`,
+
+  /** Severity Icon classes */
+  iconContainer: `${notificationPrefix}__icon-container`,
+  serverityIcon: `${notificationPrefix}__serverity-icon`,
```
Comment (travor20814) [inline packages/core/src/notification/notification.ts:28]: typo: serverity => severity

File: packages/core/src/notification/_notification.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@use '~@mezzanine-ui/system/severity';
+
+$prefix: mzn-notif;
+$root-prefix: #{$prefix}-root;
+$severities: severity.$severities-for-feedback;
```
Comment (travor20814) [inline packages/core/src/notification/_notification.scss:5]: renamed after rebase `main` branch
`$severities-for-feedback` => `$severities-wth-info`

**Review by travor20814 — CHANGES_REQUESTED** [review]
Some types are broken.

截圖 2021-03-29 下午6 08 31

**Comment by Kuanisinajar** [comment]
2021/05/11:
- refactor: `Notifier` configs extensible
- feat: `Notifier` has `getConfig` method
- refactor: `Message` add slide-fade transition

**Review by travor20814 — CHANGES_REQUESTED** [review]

**Review by travor20814 — APPROVED** [review]

**Pull request merged**